visa_mock/base/base_mocker.py

```diff
@@ -223,3 +223,7 @@ def send(self, scpi_string: str) -> Any:
 
 
 scpi = BaseMocker.scpi
+
+
+def make(scpi_string_pattern: str) -> str:
```

Comment:
Do not call this function `make` as it is very uninformative. I only gave you an example and did not intend to for you to copy-paste verbatim.

Reply:
a different name is used.